doc/user/content/integrations/_index.md

```diff
@@ -228,10 +228,10 @@ The level of support for these tools will improve as we extend the coverage of `
 
 ### Reverse ETL
 
-| Service   | Support level                    | Notes                                                                               |             |
-| --------- | -------------------------------- | ----------------------------------------------------------------------------------- | ----------- |
-| Census    | {{< supportLevel researching >}} | Not supported yet. Subscribe via "Notify Me" to register interest.                  | [](#notify) |
-| Hightouch | {{< supportLevel in-progress >}} | Connect using a [PostgreSQL source](https://hightouch.io/docs/sources/postgresql/). | [](#notify) |
+| Service   | Support level                    | Notes                                                                                                |             |
+| --------- | -------------------------------- | ---------------------------------------------------------------------------------------------------- | ----------- |
+| Census    | {{< supportLevel production >}}  | Connect using Census's [native Materialize source](https://docs.getcensus.com/sources/materialize).  |             |
```

Comment:
I'm not really comfortable with this until we test the integration, but also don't strongly oppose. This is on @dehume's list for next week.

Reply:
Ack, backed out for now.